ipaclient/plugins/dns.py

```diff
@@ -62,7 +62,7 @@ def __get_part_param(rrtype, cmd, part, output_kw, default=None):
 def prompt_parts(rrtype, cmd, mod_dnsvalue=None):
     mod_parts = None
     if mod_dnsvalue is not None:
-        name = record_name_format % rrtype.lower()
+        name = record_name_format % unicode(rrtype.lower())
```

Comment:
As you discussed with Standa, this, is a proper fix. Thanks, sorry for confusion

Reply:
@Tiboris thanks, mate!